portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -60,33 +64,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    String nickname = request.getParameter("nickname");
+    String NICKNAME = request.getParameter("nickname");
```

Comment:
Sorry I was probably confusing for using constants. I meant having NICKNAME as a class member and use that here, e.g.:

private final static String NICKNAME = "nickname";

In the function body, use it like request.getParameter(NICKNAME)

Reply:
so after I have the class member NICKNAME, would the variables inside the functions be named "nickname" ? What's the best way to do that?

Reply:
So per the Java style guide, constant variables would be named with everything capitalized. It's just a variable, but it's like a class-level global variable that you can refer to in your code. The motivation behind this is that if someday you need to change the parameter name from "nickname" to "alias", you can just change one place.